package.json

```
@@ -30,7 +30,8 @@
     "react-tooltip": "4.5.1",
     "stackdriver-errors-js": "0.12.0",
     "tss-react": "4.9.18",
-    "uuid": "11.1.0"
+    "uuid": "11.1.0",
+    "validator": "^13.15.15"
```

Minor suggestion - we usually pin dependencies exactly
```suggestion
    "validator": "13.15.15"
```

---

I wonder if we could reuse this library in many of our form component validations. See, for example, https://github.com/DataBiosphere/duos-ui/blob/develop/src/libs/utils.js#L103